### DISCUSSION TITLE
cannot set stylable value after upgrading to 0.21.7 from 0.19.5

### DISCUSSION THREAD

**Discussion by rajevar** [GrapesJS/grapesjs#5453] [Q&A]
after upgrading to the latest version of grapesJS, `component.getDefaults().stylable = ['property-1', 'property-2'] ` is not setting the value of stylable.
JSfiddle : https://jsfiddle.net/e2m4gc1f/

```
console.log("before: " + component.model.getDefaults().stylable);
component.model.getDefaults().stylable = ['width', 'height', 'display', 'padding', 'margin', 'float'];
console.log("after: " + component.model.getDefaults().stylable);
```

`grapesJS version : 0.19.5`
"before: true"
"after: width,height,display,padding,margin,float"

`grapesJS version : 0.21.7`
"before: true"
"after: true"

Is there a way to set the value of `stylable` via code ? 

Thank you,

**Comment by rajevar** [comment]
got the issue resolved by creating custom component and adding `stylable` to model defaults.
Still curious why setting stylable as mentioned in original post didn't work.

**Reply by artf** [reply]
I'm curious to know where did you find that way to extend components, if that worked before probably it was a bug 😅
